fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/network/method/message/data/election/CastVoteTest.kt

```diff
@@ -26,15 +27,21 @@ class CastVoteTest {
   private val questionId2 = generateElectionQuestionId(electionId, "Question 2")
   private val writeInEnabled = false
   private val writeIn = "My write in ballot option"
+  private val vote1 = 1
+  private val vote2 = 2
+  private val vote1Base64 = Base64URLData(vote1.toString().toByteArray()).encoded
+  private val vote2Base64 = Base64URLData(vote2.toString().toByteArray()).encoded
 
   // Set up a open ballot election
   private val plainVote1 = PlainVote(questionId1, 1, writeInEnabled, writeIn, electionId)
   private val plainVote2 = PlainVote(questionId2, 2, writeInEnabled, writeIn, electionId)
```

Comment:
Now that you defined vote1 and vote2 as variable you could also modify plainVote1/2 to use such variables instead of the hardcoded 1 and 2